nix/shell.nix

```diff
@@ -8,7 +8,7 @@ let
 
   /* Lock requires Xcode verison. */
   xcodeWrapper = callPackage ./pkgs/xcodeenv/compose-xcodewrapper.nix { } {
-      versions = ["14.3" "15.1" "15.2" "15.3" "15.4"];
+      versions = [ ];
```

Comment:
Why?

Reply:
I have kept it empty to accept whatever version of Xcode exists on CI hosts. 
Waiting to complete the Xcode migration of CI hosts and then I will pin 16.x here

Reply:
Well, if there's no known versions of Xcode that break `status-go` builds that might be fine.